src/components/useZoomPan.js

```diff
@@ -56,7 +57,17 @@ export default function useZoomPan({
       onMoveEnd({ coordinates: [x, y], zoom: d3Event.transform.k }, d3Event)
     }
 
+    function filterFunc() {
+      if (filter) {
```

Comment:
There doesn't seem to be any definition of `filter` anywhere. Did you mean to check for `filterZoomEvent` here?

Reply:
Yes, I am sorry.
I have fixed it.